tests/sandbox/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG BASE=containers.intersystems.com/intersystems/iris-community:2024.1
```

Comment:
Nit: General comment - why not store the IRIS version in one place in a file somewhere and extract it for use in various other test files? Otherwise need to find/replace every time

Reply:
I'm pretty new to docker - will look to see if there's an easy way to do that. (If you know of one off the top of your head, please tell me!)

Reply:
@isc-shuliu do you have any thoughts on this from a Docker perspective?

Reply:
On further thought: it's reasonable that we'd want to use different versions in different places, especially for the new "sandbox" container. (I think this is something it's likely a dev would briefly change locally, say to use irishealth-community or a different version instead.)

We also want to be intentional about bumping versions (generally). So the Dockerfile is a reasonable place for this.